docs/ref/databases.txt

```diff
@@ -439,6 +439,24 @@ more details.
 .. _MySQL option file: https://dev.mysql.com/doc/refman/5.6/en/option-files.html
 .. _MySQLdb documentation: http://mysql-python.sourceforge.net/
 
+.. _mysql-sql-mode:
+
+Setting ``sql_mode``
+~~~~~~~~~~~~~~~~~~~~
+
+From MySQL 5.7 onwards, the default value of the ``sql_mode`` option contains
```

Comment:
N.B. It's also the default in fresh installs of MySQL 5.6 (not on top of existing data from 5.5). See morgo's comment on https://code.djangoproject.com/ticket/15940